easybuild/easyconfigs/x/Xmipp/Xmipp-3.1-intel-2015a.eb

```diff
@@ -12,12 +12,14 @@ toolchain = {'name': 'intel', 'version': '2015a'}
 patches = ['sqlite_imports.patch', 'library_imports.patch']
 
 dependencies = [
-    ('Python', '2.7.9'),  # Make sure Python has a working SQlite in it, the install step creates a sqlite db.
-    ('freetype', '2.5.3'),
-    ('LibTIFF', '4.0.3'),
+    ('Java', '1.8.0_40', '', True),
+    ('Python', '2.7.9'),  # should have a working sqlite, and provide numpy
+    ('freetype', '2.5.5'),
+    ('LibTIFF', '4.0.4beta'),
```

Comment:
it's the latest 'stable' release according to the website

I agree the naming is shady, I wouldn't mind going with 4.0.3, but since they claim it's 'stable'...